### PR TITLE
Update contact email address in LAMMPS sources

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This changes the contact email address in the LAMMPS sources to developers@lammps.org
Also the tool for checking the LAMMPS homepage URL was extended to check for compliance and optionally fix sources.

**Author(s)**

Axel Kohlmeyer, Temple U

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

This only changes comments, so there should be no issues.

**Post Submission Checklist**

- [x] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [x] The feature has been verified to work with the conventional build system
- [x] The feature has been verified to work with the CMake based build system

### CODE REVIEW

**Review by stanmoore1 — COMMENTED** [review]
I approve

**Review by stanmoore1 — APPROVED** [review]

File: examples/COUPLE/fortran/libfwrapper.c [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* ----------------------------------------------------------------------
    LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
-   www.cs.sandia.gov/~sjplimp/lammps.html
-   Steve Plimpton, sjplimp@sandia.gov, Sandia National Laboratories
+   https://www.lammps.org/, Sandia National Laboratories
+   The LAMMPS Developers, developers@lammps.org
```
Comment (sjplimp) [inline examples/COUPLE/fortran/libfwrapper.c]: i'd prefer this line be like this everywhere:
LAMMPS development team: developers@lammps.org

**Review by sjplimp — CHANGES_REQUESTED** [review]
Hoping this tag line is easy to change

File: examples/COUPLE/fortran/libfwrapper.c (hunk 1 above)
Comment (akohlmey) [reply examples/COUPLE/fortran/libfwrapper.c]: done

**Review by sjplimp — APPROVED** [review]

**Pull request merged**